check-services.php

```diff
@@ -64,7 +64,11 @@
 $sql = 'SELECT * FROM `devices` AS D, `services` AS S WHERE S.device_id = D.device_id ' . $where . ' ORDER by D.device_id DESC';
 
 foreach (dbFetchRows($sql) as $service) {
-    // Run the polling function
-    poll_service($service);
+    // Run the polling function only if the associated device is up
+    if ($service['status'] === "1") {
```

Comment:
I actually think we should maybe do:

` if ($service['status'] === "1" || ($service['status'] === '0' && $service['status_reason'] === 'snmp')) {`

So that if the device is marked as down for snmp then it may be up for icmp still and service checks may still work. @librenms/reviewers Thoughts?